tests/Unit/Evolution/Systems/CurvedScalarWave/Worldtube/TestHelper.hpp

```diff
@@ -0,0 +1,110 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "Domain/Creators/BinaryCompactObject.hpp"
+#include "Domain/Creators/DomainCreator.hpp"
+#include "Domain/Protocols/Metavariables.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/Domain/BoundaryConditions/BoundaryCondition.hpp"
+#include "Helpers/Domain/Creators/TestHelpers.hpp"
+#include "Helpers/Domain/DomainTestHelpers.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+
+struct TestHelper {
```

Comment:
make this a namespace? Usually we do something like `TestHelpers::CurvedScalarWave::Worldtube`